pymc3/distributions/continuous.py

```diff
@@ -26,6 +36,86 @@ class UnitContinuous(Continuous):
     def __init__(self, transform=transforms.logodds, *args, **kwargs):
         super(UnitContinuous, self).__init__(transform=transform, *args, **kwargs)
 
+def get_named_nodes(graph):
+    """Get the named nodes in a theano graph 
+    (i.e., nodes whose name attribute is not None).::
+    
+        >>> a = as_tensor_variable(4., 'a')   
+        >>> b = as_tensor_variable(5., 'b') 
+        >>> c = a ** (b-1) / (0.5 * b)
+        >>> print(get_named_nodes(c))
+        {'a': TensorConstant{4.0}, 'b': TensorConstant{5.0}}
+        
+    Parameters
+    ----------
+    graph - a theano node
+    
+    Returns:
+    A dictionary of name:node pairs.
+    """
+    return _get_named_nodes(graph, {})
+
+def _get_named_nodes(graph, nodes):
+    if graph.owner == None:
+        if graph.name is not None:
+            nodes.update({graph.name:graph})
+    else:
+        for i in graph.owner.inputs:
+            nodes.update(_get_named_nodes(i, nodes))
+    return nodes
+
+
+def draw_values(items, point=None):
```

Comment:
Can we add a use example here? I'm not actually clear on what this is supposed to do